src/Dont/DontClone.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Dont;
+
+use Dont\Exception\NonCloneableObject;
+use Dont\Exception\TypeError;
+
+trait DontClone
```

Comment:
@GeeH I discussed this with @macnibblet before, and I even experienced it in first place: basically, in most scenarios, most users will import the trait via a `use` statement, and having `class Foo { use Clone; }` will be confusing, while `class Foo { use DontClone; }` is much better.